server/src/core/server/models/comment/comment.ts

```diff
@@ -1701,3 +1701,30 @@ export async function retrieveFeaturedComments(
 
   return results;
 }
+
+export async function retrieveLatestFeaturedCommentForAuthor(
+  mongo: MongoContext,
+  tenantID: string,
+  userID: string,
+  excludedCommentID: string
+) {
+  const $match: FilterQuery<Comment> = {
+    tenantID,
+    authorID: userID,
+    commentID: { $ne: excludedCommentID },
+    "tags.type": GQLTAG.FEATURED,
+    status: { $in: PUBLISHED_STATUSES },
```

Comment:
I think, since all featured comments are always published, you can remove this status filter to improve the performance of this query.

If it is ever rejected, the featured tag will also be removed.

Reply:
Okay. I can remove for `retrieveFeaturedComments` too probably then I guess (it's basically the same query, just I added in `authorID` for this one)